stable/grafana/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: grafana
-version: 5.0.13
-appVersion: 6.7.1
+version: 5.0.14
```

Comment:
Should be 5.0.15 for now. @steven-sheehy

Reply:
Done